test/unit/schema/oracle.js

```diff
@@ -497,6 +497,42 @@ describe("Oracle SchemaBuilder", function() {
     expect(tableSql[0].sql).to.equal('alter table "users" add "foo" decimal(2, 6)');
   });
 
+  it('test set comment', function() {
+    tableSql = client.schemaBuilder().table('users', function(t) {
+      t.comment('Custom comment');
+    }).toSQL();
+
+    equal(1, tableSql.length);
+    expect(tableSql[0].sql).to.equal('comment on table "users" is \'Custom comment\'');
+  });
+
+  it('test set empty comment', function() {
+    tableSql = client.schemaBuilder().table('users', function(t) {
+      t.comment('');
+    }).toSQL();
+
+    equal(1, tableSql.length);
+    expect(tableSql[0].sql).to.equal('comment on table "users" is \'\'');
+  });
+
+  it('test set empty comment using null', function() {
```

Comment:
I wouldn't consider setting comment to be empty string by setting it to null or undefined as a intended behaviour. So better to not add test for that functionality, since better test would be throwing an error because of invalid parameter.